src/factories/Legend.ts

```diff
@@ -0,0 +1,57 @@
+module n3Charts.Factory {
+  'use strict';
+
+  export class Legend extends Utils.BaseFactory {
+
+    // We might wanna find another name for this : it's not an SVG (same as for
+    // the tooltip).
+    // [lorem--ipsum]
+    private svg:D3.Selection;
```

Comment:
Hmmm it doesn't _have_ to be a div either... Too bad  `element` is already taken...